README.md

```diff
@@ -167,7 +168,7 @@ Alacritty looks for the configuration file at the following paths:
 
 ### Windows
 
-On Windows the config file is located at:
+On Windows the config file should be put in:
```

Comment:
```suggestion
On Windows the config file should be at:
```

We're directly linking a file location, not a directory. So we shouldn't use 'in'.